compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Dialog.desktop.kt

```diff
@@ -394,4 +401,39 @@ interface DialogWindowScope : WindowScope {
      * [ComposeDialog] that was created inside [androidx.compose.ui.window.DialogWindow].
      */
     override val window: ComposeDialog
+}
+
+/**
+ * Modal dialogs block all input to some windows.
+ *
+ * [DialogModalityType] defines the which set of windows input is blocked to.
+ */
+class DialogModalityType private constructor(val name: String) {
```

Comment:
I see it's used only in `ExperimentalComposeUiApi` functions. Do we want to make it stable together or we're ready to mark this one stable from the start?
If so, do we need to add non-experimental overload to actually use it without relying on experimental API?

Reply:
Made it experimental too.